pkg/apis/policy/validation/validation_test.go

```diff
@@ -389,6 +389,14 @@ func TestValidatePodSecurityPolicy(t *testing.T) {
 	invalidProcMount := validPSP()
 	invalidProcMount.Spec.AllowedProcMountTypes = []api.ProcMountType{api.ProcMountType("bogus")}
 
+	emptyAllowedCSIDrivers := validPSP()
+	emptyAllowedCSIDrivers.Spec.Volumes = []policy.FSType{policy.CSI}
+	emptyAllowedCSIDrivers.Spec.AllowedCSIDrivers = []policy.AllowedCSIDriver{}
+
+	emptyAllowedCSIDriverName := validPSP()
+	emptyAllowedCSIDriverName.Spec.Volumes = []policy.FSType{policy.CSI}
+	emptyAllowedCSIDriverName.Spec.AllowedCSIDrivers = []policy.AllowedCSIDriver{{Name: ""}}
```

Comment:
unused? Did you mean to add these to the error cases?

Reply:
Thanks, removing.